adminpages/member-edit/pmpro-class-member-edit-panel-orders.php

```diff
@@ -104,6 +105,25 @@ protected function display_panel_contents() {
 									}
 								?>
 							</td>
+							<td>
+								<?php
+								// Logic to get the gateway name and output it neatly.
+									if ( ! empty( $invoice->gateway ) ) {
+										if ( ! empty( $pmpro_gateways[$invoice->gateway] ) ) {
+											$gateway = esc_html( $pmpro_gateways[$invoice->gateway] );
+										} else {
+											$gateway = esc_html( ucwords( $invoice->gateway ) );
+										}
+										if ( $invoice->gateway_environment == 'sandbox' ) {
+											$gateway .= ' (test)';
+										}
+
+										echo $gateway;
+									} else {
+										esc_html_e( '&#8212;', 'paid-memberships-pro' );
```

Comment:
```suggestion
										echo  '&#8212;';
```
I don't think we need to localize this, a dash should be shown for all languages.